heron/healthmgr/src/java/BUILD

```diff
@@ -14,6 +14,8 @@ healthmgr_deps_files = [
     "//heron/spi/src/java:utils-spi-java",
     "//heron/spi/src/java:metricsmgr-spi-java",
     "//heron/proto:proto-java",
+    "//heron/proto:proto_packing_plan_java",
```

Comment:
The `proto_packing_plan_java` target is included in `heron/proto/BUILD:proto-java`. So `proto-java` should be sufficient? However, earlier my local build failed and bazel suggested that I add these targets. For now I have removed the targets to check if it is something to do with my local setup.